src/mutation.h

```diff
@@ -114,6 +114,9 @@ struct mutation_branch {
     // Modifier for the rate at which fatigue drops when resting.
     float fatigue_regen_modifier = 0.0f;
 
+    /** The item, if any, spawned by the mutation */
+    std::string spawn_item;
```

Comment:
Admittedly, I know nothing, but that being said: are you sure? I swiped the implementation from fake_item in bionics.cpp/bionics.h and it looks like it passes a string. Or is the bionic solution suboptimal/inadvicable?